example/France.NewEBM.json

```diff
@@ -9,42 +9,50 @@
     "Nutrients": [
         {
             "Name": "Carbs",
+            "Range": [0.0, 50.0],
             "Energy": 4.0
         },
         {
             "Name": "Fat",
+            "Range": [0.0, 50.0],
             "Energy": 9.0
         },
         {
             "Name": "Protein",
+            "Range": [0.0, 50.0],
             "Energy": 4.0
         },
         {
             "Name": "Sodium",
+            "Range": [0.0, 50.0],
             "Energy": 0.0
         }
     ],
     "Foods": [
         {
             "Name": "Carbs",
+            "Price": 0.0,
```

Comment:
Should this maybe be `null` instead? Carbs ain't free, ya know :stuck_out_tongue: